events.ics

```diff
@@ -0,0 +1,41 @@
+---
+layout: null
+---
+BEGIN:VCALENDAR
+VERSION:2.0
+PRODID:-//{{ site.title }}//Veranstaltungskalender//DE
+METHOD:PUBLISH
+X-WR-CALNAME:{{ site.name }}
+X-WR-CALDESC:Veranstaltungen der {{ site.title }}
+CALSCALE:GREGORIAN
+BEGIN:VTIMEZONE
+TZID:Europe/Berlin
+X-LIC-LOCATION:Europe/Berlin
+BEGIN:DAYLIGHT
+TZOFFSETFROM:+0100
+TZOFFSETTO:+0200
+TZNAME:CEST
+DTSTART:19700329T020000
+RRULE:FREQ=YEARLY;BYMONTH=3;BYDAY=-1SU
+END:DAYLIGHT
+BEGIN:STANDARD
+TZOFFSETFROM:+0200
+TZOFFSETTO:+0100
+TZNAME:CET
+DTSTART:19701025T030000
+RRULE:FREQ=YEARLY;BYMONTH=10;BYDAY=-1SU
+END:STANDARD
+END:VTIMEZONE
+{% for post in site.pages reversed %}{% assign sitedate = site.time | date: "%Y%m%d" %}{% assign postdate = post.date | date: "%Y%m%d" %}{% if postdate  > sitedate %}BEGIN:VEVENT
+UID:{{ post.date | date: "%Y%m%d" }}@cyberland.ijug.de
+ORGANIZER;CN="CyberLand":MAILTO:no-reply@cyberland.ijug.de
+SUMMARY:CyberLand: {{ post.title }}{% if post.speaker %} ({{ post.speaker | xml_escape }}{% if post.speaker2 %}, {{ post.speaker2 | xml_escape }}{% endif %}{% if post.speaker3 %}, {{ post.speaker3 | xml_escape }}{% endif %}){% endif %}
```

Comment:
Da sind auch noch JUG Darmstadt spezifische Inhalte drin. Speaker gibt es bei den CyberLand Events nicht, nur bei den Sessions.

Reply:
Behoben in 44e557f0a4f6ac2a7711614dab946af3da0844bf